### PR TITLE
Updated README.md to reflect correct packages for Ubuntu.

### PR DESCRIPTION
**Pull request by lyndonhenry**
This is an update to the list of required packages for Ubuntu in README.md as per issue #424.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,23 @@ Follow the steps below to compile and install Soufflé on a UNIX system:
...
+	flex \
+	g++ \
+	libsqlite3-dev \
+	libtool \
+	libncurses5-dev \
+	lsb-release \
+	make \
+	openjdk-8-jdk \
+	git \
+	zlib* \
```
Comment (mmcgr) [inline README.md]: Why not just zlib1g-dev? #424 suggested the missing libncurses5-dev library was the problem. zlib* can pull in more than is needed, regardless.

File: README.md [hunk 2]
```diff
@@ -47,7 +47,23 @@ Follow the steps below to compile and install Soufflé on a UNIX system:
 1.  Install the needed dependencies. G++ 4.8 or greater is recommended to compile Soufflé. openjdk-7 can be used if openjdk-8 is not available on your platform.
 
 ```
-sudo apt-get install autoconf automake bison build-essential doxygen flex g++ git libsqlite3-dev libtool make openjdk-8-jdk pkg-config python zlib1g-dev
+apt-get install -y build-essential;
```
Comment (lyndonhenry) [inline README.md]: I've split it into two commands as the first set of packages is matched name for name in most yum based package repositories I've looked at. For example it works (replacing `apt-get` with `yum`) on Amazon Linux, which maintains compatibility with Red Hat Enterprise Linux 6. My thinking is that this provides a better starting point for people to try and install Souffle's dependencies on systems we do not provide a lot of support for. As we test on different systems, we can add further instructions for them, still with the advantage of this fall back.

 The -y should be removed, I agree it should be up to the user.

File: README.md (hunk 1 above)
Comment (lyndonhenry) [inline README.md]: This is true, I'll replace zlib* with zlib1g-dev.

**Comment by lyndonhenry** [comment]
Full list of changes in README.md, after review:
- Kept existing lexicographical ordering for package list, so that it is more readable and maintainable. 
- Removal of -y flag for install, as the decision to install should be up to the user.
- Replacement of zlib* glob with more specific zlib1g-dev, to avoid unnecessary dependencies.
- Removal of ; character at end of line from package installation list, for better readability.
- Addition of pkg-config to dependencies, for libc6, libglib2.0-0, and libpopt0 libraries.

**Pull request merged**